jar-service/src/main/java/com/sixsq/slipstream/module/ImageFormProcessor.java

```diff
@@ -180,7 +180,7 @@ private void parseTargets(Form form) throws ValidationException {
 
 		Set<Target> targets = new HashSet<Target>();
 
-		for (String targetName : TARGET_SCRIPT_NAMES) {
```

Comment:
I think the definition of `TARGET_SCRIPT_NAMES` should be removed (it's at the top of this file).